.eslintrc.json

```diff
@@ -1,6 +1,8 @@
 {
-  "extends": "next/core-web-vitals",
+  "extends": ["next/core-web-vitals", "plugin:@typescript-eslint/recommended"],
   "rules": {
-    "no-constant-binary-expression": "error"
+    "no-constant-binary-expression": "error",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-unused-vars": "off"
```

Comment:
I'll fix those `off` rules over time and turn it back on later. Especially `no-unused-vars` should be good to go before the release.